### ISSUE TITLE
Wireshark - Download URL

### ISSUE THREAD

**Issue by vivalter** [Installomator/Installomator#1683]
Hello! 

There seems to be an error when updating Wireshark. 

I am using the most recent Installomator version 10.6beta (2024-04-23)

The script exited with code 1 because it could not find the required 'downloadURL' parameter.

It seems to have changed with the newest version of Wireshark.

PR #1343 had a similar issue in the past. 

Thank you in advance!

```
Executing Policy Installomator - Wireshark
Running script Installomator - 10.6beta (2024-04-23)...
Script exit code: 1
Script result: 2024-06-04 10:05:15 : REQ   :  : shifting arguments for Jamf
2024-06-04 10:05:15 : REQ   : wireshark : ################## Start Installomator v. 10.6beta, date 2024-04-23
2024-06-04 10:05:15 : INFO  : wireshark : ################## Version: 10.6beta
2024-06-04 10:05:15 : INFO  : wireshark : ################## Date: 2024-04-23
2024-06-04 10:05:15 : INFO  : wireshark : ################## wireshark
2024-06-04 10:05:15 : INFO  : wireshark : setting variable from argument NOTIFY=silent
2024-06-04 10:05:15 : INFO  : wireshark : setting variable from argument BLOCKING_PROCESS_ACTION=ignore
2024-06-04 10:05:15 : ERROR : wireshark : need to provide 'downloadURL'
Error running script: return code was 1.
```

**Comment by Jack7559** [comment]
Hey If you are an ubuntu user you can use the following Solution : https://askubuntu.com/questions/612654/how-to-fix-apt-get-broken-updating-wireshark 

It is telling you to purge the previous version and freshly install the latest version

**Comment by acodega** [comment]
This is now fixed in the [10.6 release](https://github.com/Installomator/Installomator/releases/tag/v10.6).
